commit 5fb936786601a1cd013a5d436adde65982b1e13c
Author: Jia Tan <jiat0218@gmail.com>
Date:   Mon Mar 6 21:37:45 2023 +0800

    xz: Add warning if Capsicum sandbox system calls are unsupported.
    
    The warning is only used when errno == ENOSYS. Otherwise, xz still
    issues a fatal error.

diff --git a/src/xz/file_io.c b/src/xz/file_io.c
index 71b5377e..1a7dac13 100644
--- a/src/xz/file_io.c
+++ b/src/xz/file_io.c
@@ -238,6 +238,8 @@ capsicum_error:
 	// implement the capability system calls, then the capsicum system
 	// calls will fail and set errno to ENOSYS.
 	if (errno == ENOSYS) {
+		message_warning(_("%s: Cannot enable the sandbox"),
+				strerror(errno));
 		sandbox_allowed = false;
 		return;
 	}